pom.xml

```diff
@@ -589,6 +590,12 @@
         </exclusions>
       </dependency>
 
+      <dependency>
```

Comment:
I would place this dependency management in the "zeppelin-server/pom.xml" to keep the parent pom.xml clear.

Reply:
Basically, I agree with it but it's related to `markdown` and `zeppelin-server`. In my understanding, it needs to add here to affect them in my understanding.

Reply:
I'm fixing not to set dependency management block on pom.xml directly. :-)